blog/2023-03-16-financial-revolution-powered-by-the-blockchain/financial-revolution-powered-by-the-blockchain.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Financial Revolution powered by blockchain Technology"
+description: In this article,we will look at how blockchain technology can be leveraged to reimagine money to create the condition for prosperity for everyone .
+authors:
+  - name: Ronald Nzioki
+    title:Technical Writer
+    url: https://github.com/rony23-byte
+    image_url: https://github.com/rony23-byte.png
```

Comment:
Kindly use this link for the image [URL](https://avatars.githubusercontent.com/u/68056049?v=4)